datasets/ubuntu_dialogs_corpus/ubuntu_dialogs_corpus.py

```diff
@@ -1,13 +1,13 @@
 """TODO(ubuntu_dialogs_corpus): Add a description here."""
 
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
+from __future__ import absolute_import, division, print_function
```

Comment:
is there a problem with black smh ?

Reply:
I don't know whether that's `black` or `isort`, but I don't think it's a problem. 

It's just that we don't check quality on the `datasets` folder at the moment - see the .circle_ci folder, but I added `./datasets` to the `Make style` file. So I think it's just that some people upload datasets without formatting them and I correct it :D

Reply:
IMO, we should add `./datasets` to the circle ci quality check!